env.yml

```diff
@@ -0,0 +1,28 @@
+name: eddymotion
+channels:
+  - conda-forge
+# Update this ~yearly; last updated Jan 2024
+dependencies:
+  - python=3.12
+  # Intel Math Kernel Library for numpy
+  - mkl=2023.2.0
+  - mkl-service=2.4.0
+  # git-annex for templateflow users with DataLad superdatasets
+  - git-annex=*=alldep*
+  # Workflow dependencies: ANTs
+  - ants=2.5
+  - pip
+  - pip:
+    - datalad
+    - dipy>=1.3.0
+    - h5py
+    - joblib
+    - tox
+    - tox-gh-actions
+    - coverage
+    - pytest
+    - pytest-cov
+    - pytest-env
+    - pytest-xdist
```

Comment:
Unrelated to this PR, but wondering whether all dependencies in `pyproject.toml` should be listed here.